styleguide/_themes/derek/scss/components/rsTabs.scss

```diff
@@ -0,0 +1,126 @@
+.rsTabs {
+  display: block;
+  margin-bottom: 10px;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+}
+
+.rsTabs-menu {
+  list-style: none;
+  margin: 0;
+  margin-left: 15px;
+  padding: 0;
+
+  .rsTabs-tabActive {
+    background-color: $rackspace-red;
+    border: 1px solid $rackspace-red;
+    padding-top: 15px;
+    position: relative;
+    z-index: 5;
+
+    &:hover {
+      background-color: $rackspace-red;
+    }
+
+    .rsTabs-tabLink {
+      color: $white;
+    }
+  }
+}
+
+.rsTabs-tabItem {
+  border: 1px solid $gray-light;
+  border-bottom: 1px solid transparent;
+  display: inline-block;
+  margin-right: -4px;
+  padding: 10px 25px;
+
+  &:not(:first-child) {
+    border-left: 1px solid transparent;
+  }
+
+  &:hover {
+    background-color: $gray-lighter;
+    cursor: pointer;
+  }
+}
+
+.rsTabs-tabLink {
+
+  &:hover {
+    text-decoration: none;
+  }
+
+  &:focus {
+    text-decoration: none;
+  }
+}
+
+.rsTabs-contentContainer {
+  border-bottom: 2px solid;
+  border-color: $rackspace-red;
+  border-top: 2px solid;
+  min-height: 10px;
+  padding: 10px 0;
+
+  .rsTabs-tabActive {
+    display: block;
+  }
+}
+
+[class^='rsTabs-tabContent-'] {
+  display: none;
+}
```

Comment:
This going to be a performance issue.  I'd recommend having two classes on the tabs: 

`rsTabs-tabContent` and then `rsTabs-tabContent-1`. 
`rsTabs-tabContent` would have the `display: none` set and then just let JS handle the display of it.